### PR TITLE
[4.0] Change width and add legend in com_workflow

### PR DESCRIPTION
**Pull request by rjharishabh**
Pull Request for Issue #.

### Summary of Changes
Add `legend` and `col-md-6` class

### Testing Instructions
`Dashboard` > `Content` > `Workflows` > `Click on any workflow`

### Actual result BEFORE applying this Pull Request
before-workflow

### Expected result AFTER applying this Pull Request
after-workflow

### Documentation Changes Required
None

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 90ebbc8847a7da420b223ccb6b2acce0de1d0db7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33492">issues.joomla.org/tracker/joomla-cms/33492</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 90ebbc8847a7da420b223ccb6b2acce0de1d0db7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33492">issues.joomla.org/tracker/joomla-cms/33492</a>.</sub>

File: administrator/components/com_workflow/tmpl/workflow/edit.php [hunk 1]
```diff
@@ -54,25 +54,28 @@
...
 
 		<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'description', Text::_('COM_WORKFLOW_DESCRIPTION'));?>
-		<div class="row">
-			<div class="col-lg-9">
-				<div class="card card-block">
-					<div class="card-body form-vertical">
-					<?php echo $this->form->renderField('description'); ?>
+		<fieldset class="options-form">
+			<legend><?php echo Text::_('COM_WORKFLOW_DESCRIPTION'); ?></legend>
+				<div class="row">
```
Comment (richard67) [inline administrator/components/com_workflow/tmpl/workflow/edit.php]: Question on code style: Why is this and the following lines intended one tab more than the previous line with the legend? They are not inside the legend so they should be intended the same as the legend.

Comment (rjharishabh) [reply administrator/components/com_workflow/tmpl/workflow/edit.php]: Sorry my mistake

Comment (richard67) [reply administrator/components/com_workflow/tmpl/workflow/edit.php]: No problem, and no need to say sorry. If I had to say sorry for every little "mistake" I make, I had nothing else to do all day long ;-)

**Comment by richard67** [comment]
Previous test results are still valid since the commit which invalidated them was just code style (indent). I've restored them in the issue tracker so they are properly counted again.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33492">issues.joomla.org/tracker/joomla-cms/33492</a>.</sub>

**Comment by chmst** [comment]
No sorry, this is not quite correct.
Two times "Description" is too much. And it looks as if state and default are connected to the description, but they are attributes of the workflow.

**Comment by rjharishabh** [comment]
Is it good to remove `Name (en-GB)` description or the other

**Comment by richard67** [comment]
Back to pending due to comments above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33492">issues.joomla.org/tracker/joomla-cms/33492</a>.</sub>

**Comment by brianteeman** [comment]
@rjharishabh think about it. why do you think it is there. Was there a purpose in displaying that or was someone crazy and just decided to spit it out? When you work out the reason then you will have your answer.

**Comment by richard67** [comment]
@rjharishabh Now after PR #33100 has been merged, your PR here conflicts in file `administrator/components/com_workflow/tmpl/workflow/edit.php`. You could try to solve them, and if you would need help you could notify me and I could help, but the question is if it still needs this PR here now. Could you check and report back here? Thanks in advance.

**Comment by chmst** [comment]
Explaning my comment: 

grafik

A fieldset is a container around fields which belong together. So users with assistive technologies are supported. In the screen above you see, that the fieldset keeps the note, which is a description, together with state and the switcher for default. This would make the user think that the description is published / unpublished or set default. 
So please remove the additional fieldset.

**Comment by richard67** [comment]
> So please remove the additional fieldset.

... and fix the merge conflict. Or if easier, replace this PR by a new one.

**Comment by rjharishabh** [comment]
I will fix it

**Comment by rjharishabh** [comment]
Closing with a new PR #33581

**Pull request closed without merging**